recipe/recipe.yaml

```
@@ -136,6 +138,8 @@ outputs:
             - libcusolver-dev
             - libcusparse-dev
             - libnvjpeg-dev
+        - libavif
+        - libheif
```

```suggestion
        - libheif
        - libheif * lgpl*
```

does this syntax work?

historically we have had to use "the naked dependency" for the pinnings to be picked up, then the secondary one to ensure that the build string constraints get picked up.

---

Ah, sorry, of course I've forgotten about it the very last minute.

Yes, that syntax works, though @h-vetinari has expressed preference for using `[build=lgpl*]` instead.

---

sure that works. no preference for me. just a licensing "preference" issue.